### PR TITLE
chore(stepper, stepper-item): update event names

### PR DESCRIPTION
**Pull request by asangma**
re #473 

Didn't see any related e2e tests but I can add them if need be.

### CODE REVIEW

File: src/components/calcite-stepper-item/calcite-stepper-item.tsx [hunk 1]
```diff
@@ -159,7 +159,7 @@ export class CalciteStepperItem {
     }
   }
 
-  @Listen("calciteStepperItemHasChanged", { target: "parent" })
+  @Listen("calciteStepperItemChanged", { target: "parent" })
```
Comment (paulcpederson) [inline src/components/calcite-stepper-item/calcite-stepper-item.tsx:162]: I think this should be `calciteStepperItemChange` everywhere so it's present-tense

**Comment by paulcpederson** [comment]
Find and replace `calciteStepperItemChanged` with `calciteStepperItemChange` and you should be good to go 👍

File: src/components/calcite-stepper-item/calcite-stepper-item.tsx (hunk 1 above)
Comment (asangma) [reply src/components/calcite-stepper-item/calcite-stepper-item.tsx:162]: Oh dang!
`calciteStepperWouldHaveBeenChanged`?
jk 
will do

Comment (asangma) [reply src/components/calcite-stepper-item/calcite-stepper-item.tsx]: Thanks @paulcpederson. I did the thing.

**Comment by paulcpederson** [comment]
Nice, thanks @asangma !

**Comment by asangma** [comment]
Yew!

**Pull request merged**